components/Header.tsx

```diff
@@ -5,40 +5,51 @@ import { useRouter } from 'next/router';
 import { IoLogoGithub } from 'react-icons/io';
 import { BsPeopleFill } from 'react-icons/bs';
 
-import Search from './Search';
+import Search, { SearchProps } from './Search';
 
-function Header() {
+
+const SearchBar = (props: SearchProps) => {
   const router = useRouter();
 
+  return (router.pathname === '/repos/[language]' && (
+      <Search {...props} />
+    )
+  )
+}
+
+function Header() {
   return (
-    <div className="border-b-[0.5px] border-dashed border-b-2023-manga-3 flex justify-center mb-5">
-      <div className="justify-between px-2 mb-2 ml-1.5 w-3/4 shadow-lg navbar rounded-box">
-        <Link href="/">
-          <img src="/hacktoberfest.svg" alt="Hacktoberfest" />
-        </Link>
-        {router.pathname === '/repos/[language]' && (
-          <div className="justify-center flex-1">
-            <Search />
-          </div>
-        )}
-        <div className="flex-none">
-          <Link
-            href="/contributors"
-            className="btn btn-square btn-ghost umami--click--contributors-button"
-          >
-            <BsPeopleFill size="1.5rem" color='white' title="Contributors" />
+    <header className="border-b-[0.5px] border-dashed border-b-2023-manga-3 mb-5">
+      <div className="container mx-auto px-4 py-2">
+        <div className="justify-between shadow-lg navbar">
+          <Link href="/">
+            <img src="/hacktoberfest.svg" alt="Hacktoberfest" className="h-12 w-auto sm:h-auto" />
           </Link>
-          <a
-            href="https://github.com/max-programming/hacktoberfest-projects"
-            target="_blank"
-            rel="noreferrer"
-            className="btn btn-square btn-ghost umami--click--github-button"
-          >
-            <IoLogoGithub size="1.5rem" color='white' title="GitHub" />
```

Comment:
It actually had that attribute before, maybe you removed it as were not seeing any effect with or without this in your browser environment

Reply:
I would add it back, give me sometime, but it seems like its not making any difference

Reply:
It is, I've tested on 3 different devices now - without dark mode, just the normal mode. It surely wasn't black yesterday and day-before-yesterday

Reply:
> I would add it back, give me sometime, but it seems like its not making any difference

Turn off dark mode and try again on your devices

Reply:
Got the issue, its happening in light mode, its working fine on dark mode. Will fix this. @prithvi2k2 Thanks for highlighting.

Reply:
Yes, Afaik the design language of this site is based on DARK coloring... so, the goal might be to run it in dark mode only even though client is on light or dark mode. So, defining static color to logo, irrespective of light/dark modes is the solution.

Reply:
https://github.com/max-programming/hacktoberfest-projects/pull/149/commits/b95ae80f617af25c75615c169f5e6dc6f4c00e2e

Added back color attribute in above commit @prithvi2k2